### PR TITLE
Add basics aa1 to master

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Documentation (non-breaking change which adds documentation)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no application.

**What is the new behavior (if this is a feature change)?**
- It is now possible to see the status of websites built and/or maintained by us.

**Other information**:
- The version has been set to `v1.0.0`, which marks the beginning of this application.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +29,7 @@
 
 - [Table of contents](#table-of-contents)
 - [Quick start](#quick-start)
+  - [This project requires the MDB React Pro license and its library](#this-project-requires-the-mdb-react-pro-license-and-its-library)
```
Comment (pinterid) [inline README.md]:
```suggestion
- [This project requires the MDB React Pro license and its library](#this-project-requires-the-mdb-react-pro-license-and-its-library)
```

File: src/components/molecules/StatusItem/index.jsx [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+import { MDBListGroupItem, MDBIcon } from "mdbreact";
+
+//> Additional
+// Fetch but better
+import axios from "axios";
+//#endregion
+
+//#region > URLS
+const PROXY_URL = "https://cors-anywhere.herokuapp.com/";
+////#endregion
```
Comment (pinterid) [inline src/components/molecules/StatusItem/index.jsx]:
```suggestion
//#endregion

```

File: src/Routes.js [hunk 3]
```diff
@@ -8,25 +8,13 @@ import { Route, Switch } from "react-router-dom";
 /**
  * HomePage: A basic template page
  */
-import {
-  HomePage,
-  MessagePage,
-  LoginPage,
-  ProfilePage,
-} from "./components/pages";
+import { HomePage } from "./components/pages";
```
Comment (pinterid) [inline src/Routes.js:14]: This files has no regions.

File: src/components/molecules/StatusItem/index.jsx [hunk 4]
```diff
@@ -0,0 +1,268 @@
+//#region > Imports
+//> React
+// Contains all the functionality necessary to define React components
+import React from "react";
+
```
Comment (pinterid) [inline src/components/molecules/StatusItem/index.jsx:5]:
```suggestion
```

File: src/components/molecules/StatusItem/index.jsx [hunk 5]
```diff
@@ -0,0 +1,268 @@
+//#region > Imports
+//> React
+// Contains all the functionality necessary to define React components
+import React from "react";
+
+//> MDB
+// "Material Design for Bootstrap" is a great UI design framework
+import { MDBListGroupItem, MDBIcon } from "mdbreact";
+
```
Comment (pinterid) [inline src/components/molecules/StatusItem/index.jsx:9]:
```suggestion
```

File: src/components/molecules/StatusItem/index.jsx [hunk 6]
```diff
@@ -0,0 +1,268 @@
...
+//> MDB
+// "Material Design for Bootstrap" is a great UI design framework
+import { MDBListGroupItem, MDBIcon } from "mdbreact";
+
+//> Additional
+// Fetch but better
+import axios from "axios";
+//#endregion
+
+//#region > URLS
```
Comment (pinterid) [inline src/components/molecules/StatusItem/index.jsx]:
```suggestion
//#region > Constant Variables

```

File: src/components/molecules/StatusItem/index.jsx [hunk 7]
```diff
@@ -0,0 +1,268 @@
...
+const PROXY_URL = "https://cors-anywhere.herokuapp.com/";
+////#endregion
+
+//#region > Components
+class StatusItem extends React.Component {
+  constructor(props) {
+    super(props);
```
Comment (pinterid) [inline src/components/molecules/StatusItem/index.jsx:22]:
```suggestion
    super(props);
    
```

File: src/components/molecules/StatusItem/index.jsx [hunk 8]
```diff
@@ -0,0 +1,268 @@
...
+          method: "GET",
+          headers: {
+            "content-type": "text/plain",
+          },
+        })
+          .then((response) => {
+            const status = response.status;
+            const receiveDate = new Date().getTime();
+
+            console.log(response);
```
Comment (pinterid) [inline src/components/molecules/StatusItem/index.jsx]: Is this required?

**Review by pinterid — CHANGES_REQUESTED** [review]
Looks good after a few changes.

**Review by Theralun — APPROVED** [review]
Lgtm.

**Review by pinterid — APPROVED** [review]
Looks good.

**Pull request merged**